src/app.ts

```diff
@@ -30,8 +39,42 @@ app.all("/signed-image-url", (req: express.Request, res: express.Response) => {
     profile = req.body.profile;
   }
 
-  const signedUrl = format(url, salt, profile);
-  res.send({ signedUrl });
+  const panda = getPanda();
```

Comment:
There's quite a lot going on in this express handler now. I wonder if it would be worth extracting the pan domain auth parts out into a middleware or similar? I know we're only going to be protecting this one endpoint for now so there's not a benefit in terms of re-usability, but I'm thinking more about separating concerns.

Reply:
Avoiding a Middleware for now as we have to call panda.stop, which could be passed as a part of the request I suppose? But I think how it is now is slightly cleaner for now. I think if in future we have more endpoints to protect or panda.stop isn't needed, we should switch to a Middleware